src/abstract-sql-compiler.ts

```diff
@@ -869,10 +882,47 @@ ${compileRule(definitionAbstractSql as AbstractSqlQuery, engine, true).replace(
 		const depends: string[] = [];
 		const createSqlElements: string[] = [];
 		const createIndexes: string[] = [];
+		const createComputedFunctions: string[] = [];
+		const dropComputedFunctions: string[] = [];
 
 		for (const field of table.fields) {
 			const { fieldName, references, dataType, computed } = field;
-			if (!computed) {
+			if (computed != null) {
+				if (!Array.isArray(computed) && computed !== true) {
+					let sqlType =
+						sbvrTypes[dataType as keyof typeof sbvrTypes]?.types[engine];
+					if (sqlType == null) {
+						throw new Error(
+							`Unknown data type '${dataType}' for engine: ${engine}`,
+						);
+					}
+					if (typeof sqlType === 'function') {
+						sqlType = sqlType.castType;
+					}
+					const fnSignature = `"computed_${table.name}_${fieldName}"("${table.name}" "${table.name}")`;
+					createComputedFunctions.push(`\
+CREATE FUNCTION ${fnSignature}
+RETURNS ${sqlType} AS $fn$
+	${compileRule(
```

Comment:
Question about computed fields in general, but should we worry about infinite recursion if a computed field attempts to reference itself directly or even indirectly via a different computed field?